.htaccess.default

```diff
@@ -258,6 +258,7 @@ FileETag None
   # If you do not want to use an URL suffix at all, you have to add a third
   # line to prevent URLs that point to folders from being rewritten (see #4031).
   #
+  #   RewriteCond %{REQUEST_FILENAME} !-f
```

Comment:
Ah, I didn't read properly. It says "add a third line". I replaced it. Never mind.